parts/k8s/windowsazurecnifunc.ps1

```diff
@@ -68,7 +68,7 @@ Set-AzureCNIConfig
         $KubeServiceCIDR,
         [Parameter(Mandatory=$true)][string]
         $VNetCIDR,
-        [Parameter(Mandatory=$true)][string]
+        [Parameter(Mandatory=$false)][string]
         $TargetEnvironment

```

Comment:
If this is no longer required can you set to a default value by turning the last line into
`$TargetEnvironment = ""`

Reply:
Or maybe we should always pass this and set the value differently - that would clean up the Set-AzureCNIConfig call too i think